test/e2e/l7networkpolicy_test.go

```diff
@@ -190,7 +190,7 @@ func probeL7NetworkPolicyTLS(t *testing.T, data *TestData, clientPodName string,
 			return false
 		}
 		return true
-	}, 10*time.Second, time.Second)
+	}, 25*time.Second, 5*time.Second)
```

Comment:
It takes 25s to realize the rule?
Why increasing the interval?

Reply:
> Why increasing the interval?

I misunderstood the usage of the interval. I have changed it back to 1 second.

> It takes 25s to realize the rule?

No. I found that sometimes the test got the error like this:

```
        Stderr: Connecting to apis.google.com (142.250.72.142:443)
        Connecting to developers.google.com (172.217.12.142:80)
        Connecting to developers.google.com (172.217.12.142:443)
        writing to stdout
        -                     33% |**********                      | 62661  0:00:01 ETA
        -                     33% |**********                      | 62661  0:00:03 ETA
        -                     33% |**********                      | 62661  0:00:05 ETA
        -                     33% |**********                      | 62661  0:00:07 ETA
        -                     33% |**********                      | 62661  - stalled -
        wget: download timed out
```. I think we should increase the wget timeout and the `assert.Eventually` timeout.

Reply:
perhaps we should switch to a url that returns less content or a in-cluster server

Reply:
Done.

Reply:
@tnqn Could you help double-check? IMO, the implementation is a little tricky since the HTTPS server doesn't use a valid SSL certificate and the client doesn't check the certificate. Suricata only checks the request packets containing SNI.

Reply:
It's not tricky. The implementation of L7NetworkPolicy TLS protocol doesn't and can't decrypt the request, it only inspects the unencrypted field which is SNI in this case, so whether the certificate doesn't really matter here. And it's supposed to work properly without checking if the certificate is trusted, otherwise it would require users to provide trustedCAs to using L7NetworkPolicy tls rule.

Reply:
Thanks for the detailed explanation!